pkg/config/config_test.go

```diff
@@ -44,3 +47,23 @@ func TestNetworkBlockDevice_NoDevice(t *testing.T) {
 	nbdItem := vm.NetworkBlockDevice("nbd2")
 	require.Nil(t, nbdItem)
 }
+
+func TestVirtualMachine_ValidateBlockDevices(t *testing.T) {
+	vm := &VirtualMachine{}
+
+	tmpDir := t.TempDir()
+	imagePath := filepath.Join(tmpDir, "disk.qcow2")
+	size := "1G"
+
+	cmd := exec.Command("qemu-img", "create", "-f", "qcow2", imagePath, size)
+	err := cmd.Run()
```

Comment:
Note that this will fail if qemu-img is not installed on the machine. We can improve this in a follow-up PR if this turns out to be a big issue.

Reply:
We can skip this test if qemu-img is not installed. It is about better integration with qemu, not core vfkit feature. In CI qemu-img can be installed using the package manager.